src/lib/get-version-from-path.ts

```diff
@@ -20,9 +20,10 @@ export function getVersionFromPath(path: string): string | undefined {
 		if (i === 2 && TFE_VERSION_IN_PATH_REGEXP.test(el)) {
 			return true
 		}
-		if ((i === 2 || i === 3) && VERSION_IN_PATH_REGEX.test(el)) {
+		if ((i === 2 || i === 3 || i === 4) && VERSION_IN_PATH_REGEX.test(el)) {
```

Comment:
Integration versioned paths are one step deeper than the rest of the docs.

Reply:
There's a comment above we should update as well as this code here:
> 	// version is only expected to be at index 2, or 3 in the case of TF-Plugins